Python/CMakeLists.txt

```diff
@@ -4,10 +4,17 @@ option(BUILD_PYTHON_INTERFACE "Build the Python interface" ON)
 
 find_program(PYTHON3_EXECUTABLE python3)
 
-if(BUILD_PYTHON_INTERFACE AND NOT APPLE AND PYTHON3_EXECUTABLE)
+# ugly but (NOT APPLE) won't pass though check_deps so
+if (APPLE)
+      set(NOT_APPLE FALSE)
+else()
+      set(NOT_APPLE TRUE)
+endif()
+check_deps(BUILD_PYTHON_INTERFACE NOT_APPLE PYTHON3_EXECUTABLE)
+
+if(BUILD_PYTHON_INTERFACE)
       execute_process (
-           COMMAND ${PYTHON3_EXECUTABLE} -c
-           "import site, sys; sys.stdout.write(site.getsitepackages()[0])"
+           COMMAND ${PYTHON3_EXECUTABLE} -m site --user-site
```

Comment:
Changed python to install to the user, not the system, dir, because it's easier to get permissions